### DISCUSSION TITLE
20A/1V SCT-013-020 or 10A/1V SCT-013-010 current transformers ?

### DISCUSSION THREAD

**Discussion by tj57** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#97] [General]
Can I use 20A/1V SCT-013-020 or 10A/1V SCT-013-010 current transformers with board v1.3 ?
How to calculate current_cal value for them ?

**Comment by CircuitSetup** [comment]
Yes, you would just need to sever the burden resistor jumper on the back of the meter pcb. 

Then you can calibrate the CTs to the meter using the procedure [here](https://github.com/CircuitSetup/Split-Single-Phase-Energy-Meter#current-procedure).
